java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -116,7 +117,7 @@ public void keyless_transactions_with_group_of_commands(
         Object[] expectedResult = builder.apply(transaction);
 
         Object[] results = client.exec(transaction).get();
-        assertArrayEquals(expectedResult, results);
+        assertDeepEquals(expectedResult, results, true);
```

Comment:
same question as above with the assertion method in `transactions_with_group_of_commands`